omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+
+Workflow 2: Importer in clients:
+
+- OMERO.importer or OMERO.insight: You have to be a member of the group
+  you want to import to in OMERO.importer or OMERO.insight. Login as the
+  administrator with restricted privileges and perform the import for
+  others as described in the chapter of Help documentation
+  :help:`import for others <facility-manager#import>`.
+
+- CLI: (see also the videos on import on
+  `OME YouTube channel
+  <https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ>`_)::
+
+    # Login as the Importer and sudo as the user you want to import for
+    $ bin/omero --sudo Importer -u user login
+    # Create new containers belonging to the user
+    $ bin/omero obj new Dataset name=Dataset-of-user
+    $ bin/omero obj new Project name=Project-of-user
+    # Link the containers
+    $ bin/omero obj new ProjectDatasetLink parent=Project:17 child=Dataset:13
+    # Import into created Dataset
+    $ bin/omero import ~/Desktop/CMPO1.png -T Dataset:name:Dataset-of-user
+
+.. _Workflow 3:
+
+Workflow 3: Analyst
+-------------------
+Typically, Analyst in OMERO needs to
+
+- read the data (always possible, see :ref:`Workflow 1`: Data viewer)
+- change and save the rendering settings of the images (enabled by
+  :term:`Write Data` privilege, exception is
+  Private groups, where he/she cannot save Rnd settings)
+- annotate the data (enabled by :term:`Write Data` privilege, but not
+  possible in Private groups)
+- draw and save ROIs on other users' images (enabled by :term:`Write Data`
+  privilege, but no saving in Private groups possible)
+- upload and attaching of result files to the analyzed images (enabled by
+  :term:`Write Data` privilege, except Private groups, where attaching
+  is not possible)
+- create Projects and Datasets for newly imported images in groups he/she
+  is not a member of (enabled by :term:`Write Data` privilege)
+- import new images resulting from image analysis into these containers
+  (enabled by :term:`Write Data` privilege)
+- (possibly) changing the ownership of the newly created conainers and
+  contained result images to the users (enabled by :term:`Chown` privilege)
+- upload, edit and delete official scripts usable by all OMERO users
+  (enabled by :term:`Upload Scripts` privilege)
+
+Workflow 3: Analyst in clients:
+
+- OMERO.insight or Insight-ImageJ plugin: Analyst has to be a member of the
+  group where the data is. Can draw ROIs and extract analysis results from
+  the ROIs and data in any type of group. Can save ROIs except in Private
+  groups. Can upload official scripts in OMERO.insight (any group type,
+  Analyst does not have to be a member of
+  any particular group for script upload).
+
+- OMERO.web, OMERO.insight, Insight-ImageJ plugin: Analyst can adjust
+  rendering settings and save them, upload
+  attachments with results, annotate (for example tag, key-value pairs,
+  rating, commenting). All works except in Private groups when done on
```

Comment:
"All works except in Private" "These actions are not permitted in Private groups with images belonging to others"?